Pod/Classes/Lifecycle/Behaviors/Nav-Bar-Hairline-Fade/NavBarHairlineFadeBehavior.swift

```diff
@@ -101,6 +110,12 @@ fileprivate extension NavBarHairlineFadeUpdater {
     func scrollViewDidScroll(_ scrollView: UIScrollView) {
         let shift = scrollView.contentInset.top + scrollView.contentOffset.y
         hairlineAlpha = shift.scaled(from: contentOffsetFadeRange, to: 0...1, clamped: true)
+        
+        #if DEBUG
+        if logsContentOffset {
+            print("\(NavBarHairlineFadeUpdater.self) content y-offset: \(String(format: "%\(0.3)f", shift))")
```

Comment:
```suggestion
            print("\(NavBarHairlineFadeUpdater.self) content y-offset: \(String(format: "%0.3f", shift))")
```

No need to interpolate - just put it right in the string